content/docs/getting-started/create-admin-group/index.md

```diff
@@ -12,11 +12,17 @@ concept of admins to enable this.
 
 ## Prerequisites
 
-You will need to have [installed the Roadie GitHub app](/docs/gettting-started/install-github-app/) before proceeding with this step.
+For automatic Admin group management you will need to have [installed the Roadie GitHub app](/docs/gettting-started/install-github-app/) before proceeding with this step.
+
+For manual management you should be aware of the correct usernames of the users you are assigning to admin group
 
 ## Create an admin group and user
 
-Roadie admins have special powers such as the ability to add and remove Backstage plugins. We use GitHub Teams to designate admins.
+Roadie admins have special powers such as the ability to add and remove Backstage plugins. 
```

Comment:
The special powers of admins were already mentioned in the intro, so I wonder if it makes sense to cut this paragraph and the heading "Create an admin group and user" and turn the h3's below into h2's instead. Then they'll be visible in the right sidebar, which might be useful.

Reply:
Yup, removed the h2 and modified those below h3s to be more descriptive h2s instead.